lightning/src/ln/offers_tests.rs

```diff
@@ -727,6 +727,82 @@ fn creates_and_pays_for_offer_using_one_hop_blinded_path() {
 	expect_recent_payment!(bob, RecentPaymentDetails::Fulfilled, payment_id);
 }
 
+/// Verifies that the Invoice Request Message can be retried if it fails to reach the
+/// payee on the first attempt.
+#[test]
+fn creates_and_pays_for_offer_using_one_hop_blinded_path_with_retry() {
```

Comment:
You can drop `_using_one_hop_blinded_path` as it is relevant to the behavior being tested.

Reply:
Got it!
Removed it in [420ed56](https://github.com/lightningdevkit/rust-lightning/commit/420ed565bf1fce14e96c02e05fa06a7b49e3a3ec)
Thanks!